### PR TITLE
Update doc for MSTest data-driven tests

### PR DESCRIPTION
**Pull request by Evangelink**
Update MSTest data-driven documentation pages to mention `DataRow` and `DynamicData` that are the recommended ways.

#sign-off

### CODE REVIEW

**Comment by Evangelink** [comment]
There are extra technical details that I would like to describe about `DataRow` and `DynamicData`, I have seen we do give some technical details for `DataSource` but it's not clear if this is the right page for it as we are driving the page with code example. 

Please let me know if I should update furthermore or create other pages (if so please let me know the structure you want me to follow).

**Comment by Court72** [comment]
@Mikejo5000 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Evangelink** [comment]
@Mikejo5000 In case it helps, I am the new owner of MSTest repository and I am working toward improving available documentation.

**Comment by Mikejo5000** [comment]
@Evangelink Yes, thanks, that does help! I will take a look if there's any issues that should be changed. If @AbhitejJohn adds comments, we can make those changes in this or a subsequent PR.

**Review by Mikejo5000 — APPROVED** [review]
Looks good, left two comments.

File: docs/test/how-to-create-a-data-driven-unit-test.md [hunk 1]
```diff
@@ -48,21 +44,100 @@ As an example, let's assume that you have:
 
 5. A unit test class called `MathsTests` to verify the behavior of the `Maths` class.
 
-We'll test a method in `Maths` that adds two integers using a loop:
+We will test a method in `Maths` that adds two integers using a loop:
```
[the remark this answers is not in the document]
Comment (Evangelink) [reply docs/test/how-to-create-a-data-driven-unit-test.md:47]: Thanks for the tip! I will revert these changes then.

[the remark this answers is not in the document]
Comment (Mikejo5000) [reply docs/test/how-to-create-a-data-driven-unit-test.md:47]: Thanks! They affect the Acrolinx score nowadays, too.

**Comment by Mikejo5000** [comment]
#sign-off Thanks!

**Pull request merged**